terraform.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: terraform 
+  version: 1.3.9
+  epoch: 0
+  description: Terraform is a tool for building, changing, and versioning infrastructure safely and efficiently 
+  target-architecture:
+    - all
+    - !riscv64
+  copyright:
+    - license: MPL-2.0
+      paths:
+        - "*"
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/terraform
+      branch: v${{package.version}}
+      destination: terraform
```

Comment:
use tag instead of branch and `expected-commit`?

Reply:
Good idea!